hi-ml-cpath/src/health_cpath/datasets/panda_tiles_dataset.py

```diff
@@ -76,9 +76,9 @@ def __init__(self,
         # Copy columns "left" --> "tile_x" and "top" --> "tile_y"
         # to be consistent with TilesDataset `TILE_X_COLUMN` and `TILE_Y_COLUMN`
         if TileKey.TILE_LEFT in self.dataset_df.columns:
-            self.dataset_df[TilesDataset.TILE_X_COLUMN] = self.dataset_df[TileKey.TILE_LEFT]
+            self.dataset_df = self.dataset_df.assign(tile_x=self.dataset_df[TileKey.TILE_LEFT])
```

Comment:
Should this not be `TilesDataset.TILE_X_COLUMN`?

Reply:
It didn't work :( , assign expects a param name not a string/enum :(

Reply:
In that case you should be able to do `**{TilesDataset.TILE_X_COLUMN.value: self.dataset_df[TileKey.TILE_LEFT]})`?

Reply:
Good point, will try it!